srcpkgs/octave/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'octave'
 pkgname=octave
 version=9.2.0
-revision=1
+revision=2
 build_style=gnu-configure
 configure_args="--disable-docs"
 hostmakedepends="perl gcc-fortran pkg-config"
```

Comment:
```suggestion
 hostmakedepends="perl gcc-fortran pkg-config  $(vopt_if gui qt5-host-tools)"
```

Reply:
Sure, it's the same. I'm just following the style of the existing template (which I did not originally write).

Reply:
would rather follow what most templates do than what one person did like 10 years ago

Reply:
It's done. Still waiting for someone with aarch64 to confirm this fixes #51602.